### ISSUE TITLE
Symbol issues

### ISSUE THREAD

**Issue by Danp2** [loganch/AutoIt-VSCode#140]
See discussion that starts [here](https://github.com/loganch/AutoIt-VSCode/issues/128#issuecomment-1283978761).

**Comment by Danp2** [comment]
Another anomaly that I've encountered is when a region and function have the same name. 

- If the region comes first, then only the region is shown in the outline
- If the function comes first, then both entries are correctly shown
- If the region is within the function, then both entries are correctly shown
- If the function is within the region, then only the region is shown in the outline

**Comment by Danp2** [comment]
Given the below code, which is from [here](https://github.com/Danp2/au3WebDriver) --

- $_WD_SupportedBrowsers is picked up from comment in Enum line instead of Const line
- $_WD_SupportedBrowsers is detected as Enum instead of constant
- Enums on continuation line are detected as regular variables

```autoit
Global Enum _ ; Column positions of $_WD_SupportedBrowsers
        $_WD_BROWSER_Name, _
        $_WD_BROWSER_ExeName, _
        $_WD_BROWSER_DriverName, _
        $_WD_BROWSER_64Bit, _
        $_WD_BROWSER_OptionsKey, _
        $_WD_BROWSER_LatestReleaseURL, _
        $_WD_BROWSER_LatestReleaseRegex, _
        $_WD_BROWSER_NewDriverURL, _
        $_WD_BROWSER__COUNTER

Global Const $_WD_SupportedBrowsers[][$_WD_BROWSER__COUNTER] = _
        [ _
        ["chrome", "chrome.exe", "chromedriver.exe", False, "goog:chromeOptions", "'https://chromedriver.storage.googleapis.com/LATEST_RELEASE_' &amp; StringLeft($sBrowserVersion, StringInStr($sBrowserVersion, '.') - 1)", "", '"https://chromedriver.storage.googleapis.com/" &amp; $sDriverLatest &amp; "/chromedriver_win32.zip"'], _
        ["firefox", "firefox.exe", "geckodriver.exe", True, "moz:firefoxOptions", "https://github.com/mozilla/geckodriver/releases/latest", '<a.*href="\/mozilla\/geckodriver\/releases\/tag\/(?:v)(.*?)"', '"https://github.com/mozilla/geckodriver/releases/download/v" &amp; $sDriverLatest &amp; "/geckodriver-v" &amp; $sDriverLatest &amp; (($bFlag64) ? "-win64.zip" : "-win32.zip")'], _
        ["msedge", "msedge.exe", "msedgedriver.exe", True, "ms:edgeOptions", "'https://msedgedriver.azureedge.net/LATEST_RELEASE_' &amp; StringLeft($sBrowserVersion, StringInStr($sBrowserVersion, '.') - 1) &amp; '_WINDOWS'", "", '"https://msedgedriver.azureedge.net/" &amp; $sDriverLatest &amp; "/edgedriver_" &amp; (($bFlag64) ? "win64.zip" : "win32.zip")'], _
        ["opera", "opera.exe", "operadriver.exe", True, "goog:chromeOptions", "https://github.com/operasoftware/operachromiumdriver/releases/latest", '<a.*href="\/operasoftware\/operachromiumdriver\/releases\/tag\/(?:v\.)(.*?)"', '"https://github.com/operasoftware/operachromiumdriver/releases/download/v." &amp; $sDriverLatest &amp; "/operadriver_" &amp; (($bFlag64) ? "win64.zip" : "win32.zip")'] _
        ]
```

**Comment by Danp2** [comment]
> * $_WD_SupportedBrowsers is picked up from comment in Enum line instead of Const line
> * $_WD_SupportedBrowsers is detected as Enum instead of constant
> * Enums on continuation line are detected as regular variables

I believe that these issues have been addressed in #148

**Comment by Danp2** [comment]
> Another anomaly that I've encountered is when a region and function have the same name.

Fixed in #148
